signinapp/templates/subteam.html.jinja2

```diff
@@ -1,19 +1,23 @@
 {% extends "base.html.jinja2" %}
 {% from 'bootstrap5/utils.html' import render_static, render_icon %}
-{% block title %}Subteam {{subteam.name}}{% endblock %}
+{% block title %}
+  Subteam {{ subteam.name }}
+{% endblock title %}
 {% block content %}
-<div class="container">
-  <h1>{{subteam.name}}</h1>
-  <div class="row">
-    <div class="col">
-      <div>
-        <ul>
-          {%- for member in subteam.members -%}
-            <li><a href="{{url_for('user.profile', username=member.username)}}">{{member.name}}</a></li>
-          {%- endfor -%}
-        </ul>
+  <div class="container">
+    <h1>{{ subteam.name }}</h1>
+    <div class="row">
+      <div class="col">
+        <div>
+          <ul>
+            {%- for member in subteam.members -%}
+              <li>
+                <a href="{{ url_for('user.profile', username=member.username)}}">{{ member.name }}</a>
```

Comment:
Not a blocker, but the default link styling is hard to see on the background. We should probably update the color used for links to something lighter.